base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -229,6 +245,7 @@ type virtualGroupManager struct {
 	vgParams            *virtualgrouptypes.Params
 	vgfManager          *virtualGroupFamilyManager
 	freezeSPPool        *FreezeSPPool
```

Comment:
Add an enable configuration to ensure that the checker can be turned off in abnormal situations.